@ecomplus/storefront-components/src/html/TheAccount.html

```diff
@@ -21,7 +21,7 @@ <h2 class="account__greetings">
 
     <ul class="account__nav nav nav-tabs">
       <li
-        v-for="(label, i) in [i19registration, i19orders, i19favorites, i19subscriptions]"
+        v-for="(label, i) in [i19registration, i19orders, i19favorites, subscriptions.length ? i19subscriptions : null]"
```

Comment:
Na verdade acho que você teria que colocar isso aqui como um array no `data` do componente:

```suggestion
        v-for="(label, i) in navTabs"
```

Aí:

```js
data: {
  navTabs: [this.i19registration, this.i19orders, this.i19favorites],
},
```

E depois da request, quando verificado que o cliente tem pedidos recorrentes:

```js
this.navTabs.push(this.i19subscriptions)
```

Reply:
Faz sentido?